.infrastructure/helm-chart/todoapp/templates/app/configMap.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-config
+  namespace: {{ .Values.common.namespace }}
```

Comment:
Ensure that `.Values.common.namespace` is defined in your `values.yaml` file. If it's not set, the ConfigMap might not be created in the intended namespace.